### PR TITLE
Fix import queue.Empty, fix task_done call on empty queue

### PR DESCRIPTION
**Pull request by dborenko**

### CODE REVIEW

**Comment by albertomr86** [comment]
Thank you for contributing. 
I will review it over the weekend and run some tests before releasing the final version.

**Comment by kmacpher67** [comment]
I'm not a python developer but it looks like this the key happiness for the MR: 
I'm getting this error also. Is there a best practices for using this like closing a channel or something when done or the program closes. 
                finally:
                    if not hasattr(self, 'queue'):
                        self.queue.task_done()

File: python_logging_rabbitmq/handlers_oneway.py [hunk 1]
```diff
@@ -123,6 +126,8 @@ def open_connection(self):
                 self.exchange_declared = True
 
             # Manually remove logger to avoid shutdown message.
+            rabbitmq_logger.propagate = rabbitmq_logger_propagate
+            rabbitmq_logger.setlevel(rabbitmq_logger_level)
```
Comment (albertomr86) [inline python_logging_rabbitmq/handlers_oneway.py:130]: @dborenko Can you change `setlevel` to `setLevel`? Typo (capital `L`).

File: python_logging_rabbitmq/handlers_oneway.py [hunk 2]
```diff
@@ -151,22 +156,26 @@ def message_worker(self):
...
+                        body=record,
+                        properties=pika.BasicProperties(
+                            delivery_mode=2,
+                            headers=self.message_headers,
+                            content_type=self.content_type
+                        )
                     )
-                )
+                finally:
+                    if not hasattr(self, 'queue'):
```
Comment (albertomr86) [inline python_logging_rabbitmq/handlers_oneway.py:175]: We want to call `task_done()` if there is a queue instance. Remove the `not` operator to be:

```python
if hasattr(self, 'queue'):
    self.queue.task_done()
```

**Review by albertomr86 — CHANGES_REQUESTED** [review]
Thank you for opening the PR. Please address the comments.

**Comment by albertomr86** [comment]
Changed merged by #37

**Pull request closed without merging**